.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+dist
+es
+lib
```

Comment:
Long term we may have packages that contain a `lib` dir that we want checked in, but I guess we can cross that bridge when we come to it?

Reply:
Yeah I had that thought and the same conclusion :D